test/models/task_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class TaskTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
you can keep here the same approach
 ```
 task = create(:task)
assert task.persisted?
```

Reply:
I forgot about it 🤦‍♀️ . Now it's added)